app/src/components/core/ButtonFab.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import styled from "styled-components";
+import { Colors, Typography } from "../../config";
+import getSpacing from "../../config/spacing";
+
+export const ButtonFab = styled.TouchableOpacity`
+  background: ${Colors.primaryDark};
+  border: 2px ${Colors.secondary};
+  border-radius: 60px;
```

Comment:
Definir um tamanho fixo e mudar o `border-radius`.

```suggestion
  border-radius: 50%;
```